SIGs/GRC/publications/Verifier_Governance.md

```diff
@@ -16,7 +16,7 @@ The discussion that follows will refer to the following Roles and Assets inheren
 | Role | Description and Trust Relationships |
 | :---- | :---- |
 | **Verifier System Operator (VSO)** | The Cloud Service Provider (CSP) or a similar entity that owns and/or operates the hardware and optionally the operating system on which the Verifier Service runs. |
-| **Verifier Service (VS)** | The entity that operates and bears responsibility for the secure operation of a service to assess attestations. The VS may support a single customer or multiple customers defined here as Verifier Tenants. The VS is responsible for collecting Reference Values and Endorser identities for the TEEs that it will verify. The VS trusts[^1] the VSO and may choose to minimize that trust by operating its Verifier Service Software within a TEE. |
+| **Verifier Service (VS)** | The entity that operates and bears responsibility for the secure operation of a service to assess attestations. The VS may support a single customer or multiple customers defined here as Verifier Tenants. In the predominant operational model, the VS is responsible for collecting Reference Values and Endorser Identities for the TEEs that it will verify. The VS trusts[^1] the VSO and may choose to minimize that trust by operating its Verifier Service Software within a TEE. |
```

Comment:
We talked about VS taking the responsibility for the hardware and the VT taking responsibility for the code that is hosted by the TEE. I don't think this text reflects what we discussed. Doesn't the TEE include what's inside the TEE?

Reply:
Maybe it's an ambiguity of the term TEE. Usually when we use TEE it means {SGX, TDX, SNP, CCA}. Sometimes we mean the code that's protected in the TEE. I think in these documents we use consistently use Workload for the stuff in the TEE.